shell/edit/provisioning.cattle.io.cluster/index.vue

```diff
@@ -470,7 +518,8 @@ export default {
     },
 
     clickedType(obj) {
-      const id = obj.id;
+      // we need to adjust the "id" for the ember interface for non-builtin kontainerdrivers
+      const id = obj.builtIn ? obj.id : obj.parsedDisplayName;
```

Comment:
i don't think this is needed, obj.id is already obj.parsedDisplayName when builtin

Reply:
The problem is specifically with `Open Telekom` cluster-driver (non built-in), where the `id` is something random, while the `parsedDisplayName` is the correct identifier `otccce`.

Reply:
for the non built-in kontainerdrivers though opt.id is already set to parsedDisplayName (it's the `id` from `addType`)